### PR TITLE
[TOS-1030] fix (test-case-details): added record button for mobile web

### PR DESCRIPTION
**Pull request by nixonakash01**
# Description
* Jira ; https://testsigma.atlassian.net/browse/TOS-1030
* Steps for reproduce
Navigate to mobile web application
Now create a test case
Now check the Record button for mobile web
* Actual
The record button not displaying for mobile web applications
* Expected
The record button should show for mobile web also
# Fix 
added check for isMobileWeb in ngIf

### CODE REVIEW

File: ui/src/app/components/cases/test-case-details.component.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
             *ngIf="!testCase?.deleted && false"
             [versionId]="testCase?.workspaceVersionId"
             [testCase]="testCase"></app-test-case-detail-pagination>
-          <div *ngIf="version?.workspace?.isWeb && !testCase?.deleted">
+          <div *ngIf="(version?.workspace?.isWeb || version.workspace.isMobileWeb) && !testCase?.deleted">
```
Comment (tarun-testsigma) [inline ui/src/app/components/cases/test-case-details.component.html:43]: @nixonakash01 Have you tested whether the recording works fine in mobile web ?

Comment (nixonakash01) [reply ui/src/app/components/cases/test-case-details.component.html:43]: checked it. it works in mobile web
image

Comment (tarun-testsigma) [reply ui/src/app/components/cases/test-case-details.component.html:43]: @nixonakash01 The screenshot doesn't help. Check that the recording happens properly once you click on record button in mobile web

Comment (nixonakash01) [reply ui/src/app/components/cases/test-case-details.component.html:43]: @tarun-testsigma steps are recorded, i checked it

Comment (tarun-testsigma) [reply ui/src/app/components/cases/test-case-details.component.html:43]: Ok

**Review by tarun-testsigma — APPROVED** [review]

**Pull request merged**